src/Servers/IIS/IIS/src/Core/WrappingStream.cs

```diff
@@ -107,9 +107,6 @@ public override int EndRead(IAsyncResult asyncResult)
         public override void EndWrite(IAsyncResult asyncResult)
             => _inner.EndWrite(asyncResult);
 
-        public override object InitializeLifetimeService()
```

Comment:
I think the error was just telling you to add the Obsolete attribute here as well.  You'll technically change semantics by not overriding and calling `_inner`.

Reply:
Ah I see